src/utils/getTestFrameworkName.ts

```diff
@@ -17,3 +24,19 @@ export function getTestFrameworkName(deps = {}) {
 
   return "N/A";
 }
+
+function IsNodeTestingUsed() {
+  const [majorVersion] = process.versions.node.split(".").map(Number);
+  if (majorVersion <= 18) {
+    return false;
+  }
+
+  const nodeTestRegex = /(?:node\s+--test|tsx)/;
+
+  const packageJsonPath = path.join(process.cwd(), "package.json");
```

Comment:
We should not parse package.json again, scripts should be accessible from here:
https://github.com/dashlog/core/blob/6159d526453b209542bbc7df434d3637f4c34f1c/src/services/repository.ts#L155-L164